integration_tests/models/sql/schema.yml

```diff
@@ -182,3 +182,29 @@ models:
     tests:
       - dbt_utils.equality:
           compare_model: ref('data_deduplicate_expected')
+
+  - name: test_not_empty_string_failing
+    columns:
+      - name: string
+        tests:
+          - dbt_utils.not_empty_string:
+              trim_whitespace: True
+              config:
+                severity: error
+                error_if: "<1"
```

Comment:
Should these be greater than signs?

Reply:
No, they shouldn't! This is a _very_ clever way to force the tests to check the negative and positive case. 

I really like this 🧠

Reply:
All credit goes to @dbeatty10 on this approach, but it feels good knowing that we're testing both the pass and fails.